### PR TITLE
[Channel] Documentation of Channel component

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q             | A
| ------------- | ---
| Doc fix? | no
| New docs? | yes
| Fixed tickets | -

### CODE REVIEW

File: components/Channel/interfaces.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Model Interfaces
+----------------
+
+.. _component_channel_model_channel-interface:
+
+ChannelInterface
+~~~~~~~~~~~~~~~~
+
+This interface should be implemented by every custom sale channel.
```
Comment (TheMadeleine) [inline components/Channel/interfaces.rst]: `by every custom sale **Channel**`

File: components/Channel/interfaces.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Service Interfaces
+------------------
+
+.. _component_channel_model_channels-aware-interface:
+
+ChannelsAwareInterface
+~~~~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by a service responsible of managing multiple channels.
```
Comment (TheMadeleine) [inline components/Channel/interfaces.rst]: by service**s** responsible **for**

File: components/Channel/interfaces.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+.. _Sylius API ChannelAwareInterface: http://api.sylius.org/Sylius/Component/Channel/Model/ChannelAwareInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_channel_model_channels-aware-interface:
+
+ChannelsAwareInterface
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (pjedrzejewski) [inline components/Channel/interfaces.rst]: Actually, this is a model interface too. It is same as ChannelAwareInterface, it is for models associated with multiple channels. :)

File: components/Channel/index.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Channel
+=======
+
+Sale channel management implementation in PHP.
```
Comment (TheMadeleine) [inline components/Channel/index.rst]: channel**s**

**Comment by pjedrzejewski** [comment]
Thanks Jasiek! ;)

**Pull request merged**